ui/components/app/permission-cell/permission-cell.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import Box from '../../ui/box';
+import {
+  AlignItems,
+  Color,
+  IconColor,
+  JustifyContent,
+  Size,
+  TextColor,
+  TextVariant,
+} from '../../../helpers/constants/design-system';
+import {
+  AvatarIcon,
+  Icon,
+  ICON_NAMES,
+  ICON_SIZES,
+  Text,
+} from '../../component-library';
+import { formatDate } from '../../../helpers/utils/util';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+import Tooltip from '../../ui/tooltip';
+
+const PermissionCell = ({
+  title,
+  description,
+  weight,
+  avatarIcon,
+  dateApproved,
+  revoked,
+}) => {
+  const t = useI18nContext();
+
+  let iconColor = Color.primaryDefault;
+  let iconBackgroundColor = Color.primaryMuted;
+  let infoIconColor = IconColor.iconMuted;
+  let infoIcon = ICON_NAMES.INFO;
+
+  if (!revoked && weight === 1) {
+    iconColor = Color.warningDefault;
+    iconBackgroundColor = Color.warningMuted;
+    infoIconColor = IconColor.warningDefault;
+    infoIcon = ICON_NAMES.DANGER;
+  }
+
+  if (revoked) {
+    iconColor = Color.iconMuted;
+    iconBackgroundColor = Color.backgroundAlternative;
+  }
+
+  return (
+    <Box
+      className="permission-cell"
+      justifyContent={JustifyContent.center}
+      alignItems={AlignItems.flexStart}
+      marginLeft={4}
+      marginRight={4}
+      marginTop={2}
+      marginBottom={2}
```

Comment:
This should make the `margin` above / below each cell `8px` right? 

On the screen shots, it looks tighter than `8px` + `8px` between each cell, is there a chance this could be overwritten somewhere?

Reply:
Agreed to fix this with paddings.